apps/calendar/locales/calendar.en-US.properties

```diff
@@ -32,7 +32,9 @@ account-header=Account
 account-list-header=Accounts
 
 # advanced settings
-advanced-settings-short=Settings
+# The following string is spoken by screen readers and not shown on the
+# screen.
+advanced-settings-short-icon-button.ariaLabel=Settings
```

Comment:
Could you group the 2 strings together and change string to strings and is to are?